test/OpenSpaceToolkit/Astrodynamics/Trajectory/Propagator.test.cpp

```diff
@@ -519,59 +531,65 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Model_Propagator, AddDyna
     }
 }
 
-TEST_F(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Model_Propagator, AddManeuvers)
-{
-    // Create another maneuver with same accelerations but non-overlapping instants
-    Array<Instant> secondManeuverInstants = Array<Instant>::Empty();
-    for (const Instant& instant : defaultManeuverInstants_)
-    {
-        secondManeuverInstants.add(instant + defaultManeuver_.getInterval().getDuration() + Duration::Seconds(1.0));
-    }
-
-    const Maneuver secondManeuver = {
-        secondManeuverInstants, defaultManeuverAccelerationProfile_, gcrfSPtr_, defaultManeuverMassFlowRateProfile_
-    };
-
-    // Check adding maneuvers to a propagator that doesn't already have maneuvers
-    {
-        EXPECT_EQ(defaultPropagator_.getNumberOfCoordinates(), 6);
-        EXPECT_EQ(defaultPropagator_.getDynamics().getSize(), 2);
-
-        defaultPropagator_.addManeuver(defaultManeuver_, Interpolator::Type::Linear);
-
-        EXPECT_EQ(defaultPropagator_.getNumberOfCoordinates(), 7);
-        EXPECT_EQ(defaultPropagator_.getDynamics().getSize(), 3);
-
-        defaultPropagator_.addManeuver(secondManeuver, Interpolator::Type::Linear);
-
-        EXPECT_EQ(defaultPropagator_.getNumberOfCoordinates(), 7);
-        EXPECT_EQ(defaultPropagator_.getDynamics().getSize(), 4);
-    }
-
-    // Check adding maneuvers to a propagator that already has maneuvers
-    {
-        EXPECT_EQ(defaultPropagatorWithManeuvers_.getNumberOfCoordinates(), 7);
-        EXPECT_EQ(defaultPropagatorWithManeuvers_.getDynamics().getSize(), 3);
-
-        defaultPropagatorWithManeuvers_.addManeuver(secondManeuver, Interpolator::Type::Linear);
-
-        EXPECT_EQ(defaultPropagatorWithManeuvers_.getNumberOfCoordinates(), 7);
-        EXPECT_EQ(defaultPropagatorWithManeuvers_.getDynamics().getSize(), 4);
-    }
-
-    // Create another maneuver with same accelerations but with overlapping instants
-    Array<Instant> overlappingManeuverInstants = Array<Instant>::Empty();
-    for (const Instant& instant : defaultManeuverInstants_)
-    {
-        overlappingManeuverInstants.add(
-            instant + defaultManeuver_.getInterval().getDuration() - Duration::Seconds(1.0)
-        );
-    }
-
-    const Maneuver overlappingManeuver = {
-        overlappingManeuverInstants, defaultManeuverAccelerationProfile_, gcrfSPtr_, defaultManeuverMassFlowRateProfile_
-    };
-}
+// TEST_F(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Model_Propagator, AddManeuvers)
```

Comment:
The addManeuver method is super useful, why are we removing it?

Reply:
We're not removing the method, these tests were commented out by mistake will add them back, some of the other tests were removed as we removed the FromTabulatedDynamics method.